struct.go

```diff
@@ -55,6 +55,8 @@ func handleStructCursor(cursor clang.Cursor, cname string, cnameIsTypeDef bool)
 				return clang.ChildVisit_Continue
 			}
 
+			typ.IsReturnArgument = true
```

Comment:
I do not think that every struct field is a return argument. Is there a problem you encountered?